package.json

```diff
@@ -107,7 +107,9 @@
     "react-native-svg": "^15.10.1",
     "react-native-web": "~0.19.13",
     "react-native-webview": "13.12.5",
-    "tailwindcss": "^3.4.17"
+    "scheduler": "^0.25.0",
```

Comment:
Is `scheduler` being used?

Reply:
its a peer dependency of [use-context-selector](https://www.npmjs.com/package/use-context-selector)

Reply:
Oh, got it!